.github/workflows/main.yml

```diff
@@ -55,6 +49,32 @@ jobs:
     - name: Run tests
       run: tox -e py
 
+    - name: Upload coverage report
+      run: bash <(curl -s https://codecov.io/bash) -cF tests
```

Comment:
Could you try without `-cF`? It seems that’s only meant to be used when different programming languages are involved.

Reply:
What I understand is that is just a flag, the "python" and "javascript" examples in [the docs](https://docs.codecov.io/docs/merging-reports) are just examples.

```
-F flag      Flag the upload to group coverage metrics

             -F unittests        This upload is only unittests
             -F integration      This upload is only integration tests
             -F ui,chrome        This upload is Chrome - UI tests
```
That said, I tried a few alternatives and none worked: #23, #25

Reply:
:disappointed: